File: 01.JAVA/pg-226-230/pg_228_23.java
import java.util.Scanner;

public class pg_228_23
{
    public static void main (String[] args)
    {
        Scanner enter = new Scanner(System.in);
        int num, dig;

        System.out.println("");
        System.out.println("************************************************************************************");
        System.out.println("READ AN INTEGER NUMBER AND DETERMINE IF THE SUM OF IT'S DIGITS IS A PRIME NUMBER TOO");
        System.out.println("************************************************************************************");
        System.out.println("");

        System.out.print("Write an integer number java: ");
        num = enter.nextInt();

        if(num < 0)
        {
            num = num * (-1);
        }

        int count = 0;

        while(num > 0)
        {
            dig = num % 10;
            count += dig;
            num /= 10;
        }

        boolean es_primo = true;

        if(count <= 1) 
        {
            es_primo = false;
        }
        else
            {
                int i = 2;

                while (i * i <= count)
                {
                    if (count % i == 0)
                    {
                        es_primo = false;
                    }
                    i += 1;
                }
                if (es_primo)
                {
                    System.out.println("The sum of the digits is " + count + " and it's a prime number.");
                }
                else
                {
                    System.out.println("The sum of the digits is " + count + " and it isn´t a prime number.");
                }

            }
        }
}

File: 02.PYTHON/pg-226-230/pg_228_23.py
print("")
print("***********************************************************************************")
print("READ AN INTEGER NUMBER AND DETERINE IF THE SUM OF IT'S DIGITS IS A PRIME NUMBER TOO")
print("***********************************************************************************")
print("")

num = int(input("Write an integer number py: "))
count = 0

num = abs(num)

while(num > 0):
    dig = num % 10
    count += dig
    num //= 10

es_primo = True

if (count <= 1):
    es_primo = False
else:
    i = 2
    while (i * i <= count):
        if (count % i == 0):
            es_primo = False
            break
        i += 1

    if (es_primo):
        print("The sum of the digits is ", count, " and it's a prime number.")
        
    else:
        print("The sum of the digits is ", count, " and it isn´t a prime number.")